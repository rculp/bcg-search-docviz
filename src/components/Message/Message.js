import { Message } from 'semantic-ui-react';
import './Message.scss';

export default Message;
