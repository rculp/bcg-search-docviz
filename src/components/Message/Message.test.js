import { shallow } from 'enzyme';

import React from 'react';

import Message from './Message';

describe('Message', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Message />
    );
    expect(component).toMatchSnapshot();
  });
});