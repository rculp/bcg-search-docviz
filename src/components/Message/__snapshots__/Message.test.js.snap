// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders and matches our snapshot 1`] = `
<div
  className="ui message"
/>
`;
